test/fixtures/stacks/remote/terragrunt.stack.hcl

```diff
@@ -0,0 +1,14 @@
+locals {
+	version = "v0.6.0"
+}
+
+unit "app1" {
+	source = "github.com/gruntwork-io/terraform-google-sql.git//modules/cloud-sql?ref=${local.version}"
```

Comment:
Why are we using this for our tests? Also, this isn't a unit but a module, so this is a little confusing. 

If you need to be able to reference a remote fixture in the Terragrunt repo that isn't tagged yet, you can use the commit. If the issue is squashed commits from merges, we can update the commit after the squash or merge in the fixture as a separate PR.